recipes/libevent/2.1.11/conanfile.py

```diff
@@ -0,0 +1,112 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class LibeventConan(ConanFile):
+    name = "libevent"
+    description = "libevent - an event notification library"
+    topics = ("conan", "libevent", "event")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libevent/libevent"
+    author = "Bincrafters <bincrafters@gmail.com>"
```

Comment:
Remove author

Reply:
Ok, done. Should we remove author in all following PRs? It exists in 75% of submitted recipes.